pom.xml

```diff
@@ -622,6 +622,14 @@
 					<excludeFilterFile>${project.basedir}/.spotbugs/spotbugs-exclude.xml</excludeFilterFile>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
@ppatierno I had to add this plugin to make use `Map.of()`. It is causing the build to fail. So should I remove the use `Map.of()` and use a normal Hashmap();

Reply:
@ShubhamRwt this is because `Map.of` was introduced in Java 9, so it's not about using the plugin but about the Java version. It should work even not using the plugin but updating:

```xml
<maven.compiler.source>1.9</maven.compiler.source>
<maven.compiler.target>1.9</maven.compiler.target>
```

can you try that?

Said that we should move to use the plugin and Java 11 as for the operator but this is for a different PR.

Reply:
Added the change

Reply:
You cannot change to Java 9 when we support Java 8+.

Reply:
@scholzj wdym by "we support Java 8+"? Shouldn't we also move to Java 11 as the operator?

Reply:
I think that is a discussion beyond this PR. The Bridge today supports and tests Java 8 and 11. You cannot change it to Java 9 and 11 just like that within a PR which does some tracing span modification (plus it is anyway way more complicated as you can see form the failing CI).